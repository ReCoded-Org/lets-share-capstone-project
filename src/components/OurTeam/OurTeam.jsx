import React from "react";
import Image from "next/image";
import Link from "next/link";

function OurTeam() {
    const singleProfile = {
        name: "Name Surname",
        description: "information about team member",
    };
    return (
        <div>
            <div
                id='ourTeam'
                className='flex  flex-col items-center justify-center p-5 md:flex-row'
            >
                <div
                    id='singeProfileOurTeam'
                    className=' flex h-72 w-72 flex-col items-center justify-center p-5   '
                >
                    <div className='flex h-48 w-48 cursor-pointer flex-row items-center  justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110  '>
                        <Link href='#'>
                            <Image
                                className='rounded-full'
                                src='/images/Lets-share-test-profile.png'
                                alt='Lets-share-logo'
                                width={200}
                                height={200}
                            />
                        </Link>
                    </div>
                    <div>
                        <span className=' flex flex-col justify-center p-2 text-center'>
                            <b>{singleProfile.name}</b>
                            {singleProfile.description}
                        </span>
                    </div>
                </div>
                <div
                    id='singeProfileOurTeam '
                    className=' flex h-72 w-72 flex-col items-center justify-center p-5   '
                >
                    <div className=' flex h-48 w-48 flex-row items-center justify-center  rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110'>
                        <Link href='#'>
                            <Image
                                className='rounded-full'
                                src='/images/Lets-share-test-profile.png'
                                alt='Lets-share-logo'
                                width={200}
                                height={200}
                            />
                        </Link>
                    </div>
                    <div>
                        <span className=' flex flex-col justify-center p-2 text-center'>
                            <b>{singleProfile.name}</b>
                            {singleProfile.description}
                        </span>
                    </div>
                </div>
                <div
                    id='singeProfileOurTeam flex-row md:flex-col'
                    className=' flex h-72 w-72 flex-col items-center justify-center p-5   '
                >
                    <div className='flex h-48 w-48 flex-row items-center justify-center  rounded-full bg-secondary shadow transition duration-300 ease-in-out  hover:scale-110'>
                        <Link href='#'>
                            <Image
                                className=' rounded-full'
                                src='/images/Lets-share-test-profile.png'
                                alt='Lets-share-logo'
                                width={200}
                                height={200}
                            />
                        </Link>
                    </div>
                    <div>
                        <span className=' flex flex-col justify-center p-2 text-center'>
                            <b>{singleProfile.name}</b>
                            {singleProfile.description}
                        </span>
                    </div>
                </div>
            </div>
            <div
                id='ourTeam'
                className='flex  flex-col items-center justify-center p-5 md:flex-row'
            >
                <div
                    id='singeProfileOurTeam'
                    className=' flex h-72 w-72 flex-col items-center justify-center p-5   '
                >
                    <div className='flex h-48 w-48 cursor-pointer flex-row items-center  justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110  '>
                        <Link href='#'>
                            <Image
                                className='rounded-full'
                                src='/images/Lets-share-test-profile.png'
                                alt='Lets-share-logo'
                                width={200}
                                height={200}
                            />
                        </Link>
                    </div>
                    <div>
                        <span className=' flex flex-col justify-center p-2 text-center'>
                            <b>{singleProfile.name}</b>
                            {singleProfile.description}
                        </span>
                    </div>
                </div>
                <div
                    id='singeProfileOurTeam '
                    className=' flex h-72 w-72 flex-col items-center justify-center p-5   '
                >
                    <div className='flex h-48 w-48 flex-row items-center justify-center  rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110'>
                        <Link href='#'>
                            <Image
                                className='rounded-full'
                                src='/images/Lets-share-test-profile.png'
                                alt='Lets-share-logo'
                                width={200}
                                height={200}
                            />
                        </Link>
                    </div>
                    <div>
                        <span className=' flex flex-col justify-center p-2 text-center'>
                            <b>{singleProfile.name}</b>
                            {singleProfile.description}
                        </span>
                    </div>
                </div>
            </div>
        </div>
    );
}
export default OurTeam;
