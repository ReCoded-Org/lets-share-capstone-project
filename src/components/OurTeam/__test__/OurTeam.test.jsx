import renderer from "react-test-renderer";
import OurTeam from "@/components/OurTeam/OurTeam";

it("renders correctly", () => {
    const tree = renderer.create(<OurTeam />).toJSON();
    expect(tree).toMatchSnapshot();
});
