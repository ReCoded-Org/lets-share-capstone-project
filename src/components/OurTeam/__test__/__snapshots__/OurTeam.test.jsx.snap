// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="flex flex-col items-center justify-center p-5 md:flex-row"
    id="ourTeam"
  >
    <div
      className=" flex h-72 w-72 flex-col items-center justify-center p-5 "
      id="singeProfileOurTeam"
    >
      <div
        className="flex h-48 w-48 cursor-pointer flex-row items-center justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110 "
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27200%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Lets-share-logo"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            onMouseEnter={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Lets-share-logo"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              onMouseEnter={[Function]}
              src="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75"
              srcSet="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span
          className=" flex flex-col justify-center p-2 text-center"
        >
          <b>
            Name Surname
          </b>
          information about team member
        </span>
      </div>
    </div>
    <div
      className=" flex h-72 w-72 flex-col items-center justify-center p-5 "
      id="singeProfileOurTeam "
    >
      <div
        className=" flex h-48 w-48 flex-row items-center justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27200%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Lets-share-logo"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            onMouseEnter={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Lets-share-logo"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              onMouseEnter={[Function]}
              src="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75"
              srcSet="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span
          className=" flex flex-col justify-center p-2 text-center"
        >
          <b>
            Name Surname
          </b>
          information about team member
        </span>
      </div>
    </div>
    <div
      className=" flex h-72 w-72 flex-col items-center justify-center p-5 "
      id="singeProfileOurTeam flex-row md:flex-col"
    >
      <div
        className="flex h-48 w-48 flex-row items-center justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27200%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Lets-share-logo"
            className=" rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            onMouseEnter={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Lets-share-logo"
              className=" rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              onMouseEnter={[Function]}
              src="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75"
              srcSet="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span
          className=" flex flex-col justify-center p-2 text-center"
        >
          <b>
            Name Surname
          </b>
          information about team member
        </span>
      </div>
    </div>
  </div>
  <div
    className="flex flex-col items-center justify-center p-5 md:flex-row"
    id="ourTeam"
  >
    <div
      className=" flex h-72 w-72 flex-col items-center justify-center p-5 "
      id="singeProfileOurTeam"
    >
      <div
        className="flex h-48 w-48 cursor-pointer flex-row items-center justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110 "
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27200%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Lets-share-logo"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            onMouseEnter={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Lets-share-logo"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              onMouseEnter={[Function]}
              src="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75"
              srcSet="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span
          className=" flex flex-col justify-center p-2 text-center"
        >
          <b>
            Name Surname
          </b>
          information about team member
        </span>
      </div>
    </div>
    <div
      className=" flex h-72 w-72 flex-col items-center justify-center p-5 "
      id="singeProfileOurTeam "
    >
      <div
        className="flex h-48 w-48 flex-row items-center justify-center rounded-full bg-secondary shadow transition duration-300 ease-in-out hover:scale-110"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27200%27/%3e"
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="Lets-share-logo"
            className="rounded-full"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            onMouseEnter={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="Lets-share-logo"
              className="rounded-full"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              onMouseEnter={[Function]}
              src="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75"
              srcSet="/_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-test-profile.png&w=640&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span
          className=" flex flex-col justify-center p-2 text-center"
        >
          <b>
            Name Surname
          </b>
          information about team member
        </span>
      </div>
    </div>
  </div>
</div>
`;
